content/SCALECLIReference/Auth/CLIApiKey.md

```diff
@@ -10,6 +10,193 @@ tags:
 
 {{< toc >}}
 
+## API_Key Namespace
+
+The *api_key* namespace has five commands and is based on API key management functions found in the SCALE API and web UI.
+It provides access to API kay creation and management methods through the **api_key** commands.
+
+## API_Key Commands
+
+The following **api_key** commands allow you to create, view, and delete API keys.
+
+You can enter commands from the main CLI prompt or from the snapshot namespace prompt.
+
 {{< include file="/_includes/CLIGuideWIP.md" type="page" >}}
 
-{{< taglist tag="scalecliauth" limit="10" title="Related CLI Auth Articles" >}}
\ No newline at end of file
+### Interactive Argument Editor (TUI)
+
+{{< include file="/_includes/CLI/HintInteractiveArgsEditor.md" type="page" >}}
+
+### Create Command
+
+The `create` command lets you create simple or complex API keys.
+
+{{< expand "Creating a Simple API Key" "v" >}}
+
+#### Description
+The `create` command requires the `name` property.
+Enter the command, then press <kbd>Enter</kbd>.
```

Comment:
Include how they enter the property argument, for example:
Enter a property argument using the `=` delimiter to separate property and value. 

Or if it does require the value in double quotes:
Enter a property argument using the `=` delimiter to separate property and the value double-quoted.